kpi/models/kpi_category.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
Sorry: there must have been a misunderstanding: can you use short headers similar to https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L1 including your actual information?